.python-version

```diff
@@ -1 +1 @@
-3.11.8
+3.10.11
```

Comment:
From [here](https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json) we can see that it has an arm64 build: `python-3.10.11-darwin-arm64.tar.gz`